### ISSUE TITLE
validation issue

### ISSUE THREAD

**Issue by maabdelghaffar** [titu1994/Image-Super-Resolution#11]
hello, 
i got a quick question here, 
from the epochs i have acceptable results although its not as good as what mentioned in the paper, however, i don't know why the val_PSNRLoss at last epoch is 24.xxx and when validating the model it drops to average 13.xxxx
Am i doing something wrong here ? maybe i am confused with the difference between the val psnr loss and the model validating 

thanks

Epoch 50/50
24576/24576 [==============================] - 4532s - loss: 0.0042 - PSNRLoss: 26.9141 - val_loss: 0.0055 - val_PSNRLoss: 24.2536
Validating  model
Validating  images from path 
validated image:
126058_255_RGB_part2.tif
13.5239350796
validated image:
126058_255_RGB_part4.tif
13.6472463608
validated image:
126058_255_RGB_part3.tif
14.0380191803
Average PSNR value of validation images is
13.7364002069

**Comment by titu1994** [comment]
Perhaps the validation images are not represented in the training set, therefore the model can't upscale them properly.

There can be many reasons for this. I suggest increasing training dataset images.
